tests/test_posttest.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import logging
+import os.path
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.posttest,
+    pytest.mark.topology('util')
+]
+
+
+def test_collect_techsupport(duthost):
+    """
+    A util for collecting techsupport after tests.
+
+    Since nightly test on Jenkins will do a cleanup at the beginning of tests,
+    we need a method to save history logs and dumps. This util does the job.
+    """
+    logger.info("Collecting techsupport since yesterday")
+    # Because Jenkins is configured to save artifacts from tests/logs,
+    # and this util is mainly designed for running on Jenkins,
+    # save path is fixed to logs for now.
+    TECHSUPPORT_SAVE_PATH = 'logs/'
+    out = duthost.command("show techsupport --since yesterday", module_ignore_errors=True)
```

Comment:
You can consider running `generate_dump -s yesterday` command which is less verbose. The `show techsupport` always calls `generate_dump` with verbose on under the hood: https://github.com/Azure/sonic-utilities/blob/master/show/main.py#L1201

Reply:
Thanks Xin. Updated.